make/run-cicd-script.mk

```diff
@@ -1,5 +1,9 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+GH_OWNER=mfrancisc
+GH_REPO=toolchain-cicd
+GH_BRANCH=devkeycloak
```

Comment:
reminder

Reply:
thanks! I'll remove those once I merge the cicd PR!

Reply:
done in https://github.com/codeready-toolchain/toolchain-e2e/pull/996/commits/09ae4aa053638c9e9741bae7d61b9aaf08345ac6